style.css

```diff
@@ -0,0 +1,134 @@
+
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+body {
+  font-family: 'Poppins';
+}
+
+header > div {
+    padding: 0 25px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid lightgray;
+  }
+  
+  nav {
+    width: 800px;
+  }
+  
+  nav ul {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  
+  nav a {
+    text-decoration: none;
+    color: black;
+  }
+  
+  .blackHeart {
+    width: 20px;
+    margin-right: 1rem;
+  }
+
+  .npm-logo
+  {
+    width: 55px;
+    margin-right: 1rem;
+  }
+
+  .mag-glass
+  {
+    width: 27px;
+    position: absolute;
+    left: 100px;
+    padding: 0 3px 0 0;
+    border: 2px solid rgb(177, 165, 165);
+  }
+
+  .search-button input[type="text"] 
+{
+    position:absolute;
+    left:11%;
+    top:9.5%;
+    padding: 7px;
+    width:70%;
+    background-color:white;
+    border: 2px solid rgb(177, 165, 165);
+    color:grey;
+}
+
+.sbutton
+{
+  position: absolute;
+    left:71%;
+    top:9.5%;
+    padding: 7px;
+    width:10%;
+    background-color:#fb3e44;
+    border: 2px solid rgb(177, 165, 165);
+    color:white;
+}
+
+form a
+{
+  color: black;
+  font-size: 20px;
+  text-align: center;
+  text-decoration: none;
+  padding: 1px 3px;
+}
+
+#section1 
+{
+  height: 600px;
+  background-color: rgba(232,217,217,.3);
+}
+
+#section1 > h1
+{
+  text-align: center;
+  font-size: 3.5em;
+  font-weight: bolder;
+}
+
+#section1 > article
+{
+  text-align: center;
+  
+  left: 40%;
+  font-weight: bolder;
+  }
+
+.see-plans
+{
+  position: absolute;
+    left:37%;
+    top:50%;
+    padding: 7px;
+    width:10%;
+    background-color:#fb3e44;
+    box-shadow: 8px 8px 0 rgba(251,59,73,.2);
+    border: 2px solid rgb(177, 165, 165);
+    color:white;
+}
+
+.join-free
+{
+  position: absolute;
+    left:53%;
+    top:50%;
+    padding: 7px;
+    width:10%;
+    border: 2px solid rgb(177, 165, 165);
+    color:black;
+}
+
+#section2 img
+{
+  width: 255px;
+  object-position: center;
+}
```

Comment:
Your section 3 is not style, I advised you to finish it by the end of the week, it will help you be more confident with flexbox.